packages/pds/src/db/message-queue/tables/messageQueue.ts

```diff
@@ -0,0 +1,12 @@
+import { Generated } from 'kysely'
+
+export interface MessageQueue {
+  id: Generated<number>
+  message: string
+  read: 0 | 1
```

Comment:
This is small and completely fine w/you ignoring it, but a separate "MessageQueueCursors" table might be better for two reasons -- it allows multiple separate processors (may not need it but who knows) and it would eliminate N numbers for N events, saving ... some ... space on disk

Reply:
this should still allow for multiple separate processors. we lock rows with `SELECT FOR UPDATE` that keep separate processors from stepping on each other's toes

Reply:
I'm thinking 2 processors that aren't coordinated with each other, eg maintaining their own cursors. But it's not a real concern tbh